tableau-databricks/oauth-config.xml

```diff
@@ -2,7 +2,7 @@
 <pluginOAuthConfig>
     <dbclass>databricks</dbclass>
 
-    <clientIdDesktop>602b9111-dbbc-469f-bd20-7e2db0f7f956</clientIdDesktop>
```

Comment:
what was the old clientId? where did that come from?

is that a dev/staging clientId?

Reply:
Not sure where it came from, but it doesn't work with Databricks OAuth (on any environment).